pyairtable/orm/model.py

```diff
@@ -221,30 +221,52 @@ def exists(self) -> bool:
         """
         return bool(self.id)
 
-    def save(self) -> bool:
+    def save(self, fields: Optional[List[str]] = None) -> bool:
         """
         Save the model to the API.
 
         If the instance does not exist already, it will be created;
         otherwise, the existing record will be updated.
 
+        Args:
+            fields: A list of field names to save. If provided, only these
+            fields will be updated. Otherwise, all fields will be updated.
+            The field names should be in the form the user declared them
+            in the ORM class.
+
         Returns:
             ``True`` if a record was created, ``False`` if it was updated.
         """
         if self._deleted:
             raise RuntimeError(f"{self.id} was deleted")
+
         table = self.meta.table
-        fields = self.to_record(only_writable=True)["fields"]
```

Comment:
@BAPCon what do you think of below code? If basically keeps the same logic but filter the `Model.to_record` with the list of fields to save `fields_to_save`? Also, it adds some checks to make sure that the fields to save and declared in the Model ORM.

```
assert set(fields_to_save).issubset(self._attribute_desriptor_map().keys()), "Undeclared fields in Model."
fields = self.to_record(only_writable=True)["fields"]
fields = {k:v for fields.items() if k in fields_to_save}
```

The rest of this method is the same.